util/jsonrpc-types/src/indexer.rs

```diff
@@ -48,10 +48,12 @@ impl<T> IndexerPagination<T> {
 /// SearchKey represent indexer support params
 #[derive(Deserialize)]
 pub struct IndexerSearchKey {
-    /// Script, supports prefix search
+    /// Script
     pub script: Script,
     /// Script Type
     pub script_type: IndexerScriptType,
+    /// Script search mode, optional default is `prefix`, means search script with prefix
+    pub script_search_mode: Option<IndexerScriptSearchMode>,
```

Comment:
There are only two states here, `null` == `prefix`, and I think `#[serde(default)]` is a better approach